pii_recognition/evaluation/pipeline.py

```diff
@@ -1,23 +1,28 @@
-from typing import Dict, List, Tuple, Optional
+from typing import Dict, List, Tuple
 
 from pakkr import returns
 
 from pii_recognition.evaluation.model_evaluator import ModelEvaluator
 from pii_recognition.recognisers import registry as recogniser_registry
 from pii_recognition.recognisers.entity_recogniser import EntityRecogniser
+from pii_recognition.tokenisation import tokeniser_registry
 
 
 # recogniser has been injected to meta
 @returns(recogniser=EntityRecogniser)
-def get_recogniser(
-    recogniser_name: str, recogniser_config: Optional[Dict] = None
-) -> Dict[str, EntityRecogniser]:
+def get_recogniser(recogniser_setup: Dict) -> Dict[str, EntityRecogniser]:
     recogniser_instance = recogniser_registry.create_instance(
-        recogniser_name, recogniser_config
+        recogniser_setup["name"], recogniser_setup.get("config")
     )
     return {"recogniser": recogniser_instance}
 
 
+def get_tokeniser(tokeniser_setup: Dict):
```

Comment:
Curious why `get_recogniser()` puts the recogniser into meta but this one doesn't

Reply:
It will consumed by the next step when building the evaluator and it won't be used anywhere else.

Reply:
Evaluator used to take `tokeniser_setup` dict when initialising and then build tokeniser in-place. In one of the past PRs, we've changed to have `Evaluator` takes `tokeniser` instance for initialisation.

Reply:
For consistency, you could put it in meta as well and let pakkr injects it.